staging_manifest.yml

```diff
@@ -2,17 +2,18 @@
 applications:
 - name: federalist-staging
 buildpack: https://github.com/cloudfoundry/nodejs-buildpack#v1.5.4
+stack: cflinuxfs2
 domain: fr.cloud.gov
 disk_quota: 2G
-memory: 512MB
+memory: 1G
```

Comment:
At `1G` there is no longer a memory issue. But I'm certain this was being caused by the `node-pre-gyp` installation of `node-sass`.